app/bundles/ApiBundle/EventListener/ApiSubscriber.php

```diff
@@ -4,13 +4,13 @@
 
 use Mautic\ApiBundle\Helper\RequestHelper;
 use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Translation\Translator;
```

Comment:
Why not `use Symfony\Contracts\Translation\TranslatorInterface;`?

Reply:
Because of https://github.com/mautic/mautic/blob/5.x/app/bundles/ApiBundle/EventListener/ApiSubscriber.php#L165